autoload/coverage.vim

```diff
@@ -314,4 +315,12 @@ function! coverage#EnsureProvider(provider) abort
   endif
 endfunction
 
+
+""
+" Re-define highlights after the colorscheme was changed.
+augroup coverage_highlights
```

Comment:
Autocmds should be defined in a file plugin/autocmds.vim (w/ the standard `maktaba#plugin#Enter` guard) so they're loaded in a predictable manner and can be disabled if needed.